iaso/api/mobile/entity.py

```diff
@@ -0,0 +1,106 @@
+from django.db.models import Prefetch
+from django_filters.rest_framework import DjangoFilterBackend  # type: ignore
+from rest_framework import filters
+from rest_framework import serializers
+from rest_framework.pagination import PageNumberPagination
+
+from iaso.api.common import (
+    ModelViewSet,
+    DeletionFilterBackend,
+    TimestampField,
+)
+from iaso.models import Entity, Instance, OrgUnit
+
+
+class LargeResultsSetPagination(PageNumberPagination):
+    page_size = 1000
+    page_size_query_param = "page_size"
+    max_page_size = 1000
+
+
+class MobileEntityAttributesSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Instance
+        fields = ["id", "form_id", "form_version_id", "created_at", "updated_at", "org_unit_id", "json"]
+
+    form_id = serializers.IntegerField(read_only=True, source="id")
```

Comment:
I might be wrong but this seems weird as well

Reply:
I made the change, can you check if it's ok ?

Reply:
I believe this one is OK now, yet don't forget to add a test to check it is